.github/workflows/e2e.yaml

```
@@ -14,7 +14,14 @@ concurrency:
 
 jobs:
   regular-path:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - arch: amd64
+            runner: ubuntu-latest
+          - arch: arm64
+            runner: github-hosted-ubuntu-arm64
```

According to this https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/ this image might be free/cheaper
```suggestion
          - arch: amd64
            runner: ubuntu-24.04
          - arch: arm64
            runner: ubuntu-24.04-arm
```

---

TIL Azure Cobalt 100